core/src/avm2/globals/flash/ui/keyboard.rs

```diff
@@ -38,5 +40,352 @@ pub fn create_class<'gc>(mc: MutationContext<'gc, '_>) -> GcCell<'gc, Class<'gc>
+        ("KEYNAME_SYSTEM", "Sys"),
+        ("KEYNAME_PRINT", "Print"),
+        ("KEYNAME_CLEARLINE", "ClrLn"),
+        ("KEYNAME_CLEARDISPLAY", "ClrDsp"),
+        ("KEYNAME_INSERTLINE", "InsLn"),
+        ("KEYNAME_DELETELINE", "DelLn"),
+        ("KEYNAME_INSERTCHAR", "InsChr"),
+        ("KEYNAME_DELETECHAR", "DelChr"),
+        ("KEYNAME_PREV", "Prev"),
+        ("KEYNAME_NEXT", "Next"),
+        ("KEYNAME_SELECT", "Select"),
+        ("KEYNAME_EXECUTE", "Exec"),
+        ("KEYNAME_UNDO", "Undo"),
+        ("KEYNAME_REDO", "Redo"),
+        ("KEYNAME_FIND", "Find"),
+        ("KEYNAME_HELP", "Help"),
+        ("KEYNAME_MODESWITCH", "ModeSw"),
+        ("KEYNAME_PLAYPAUSE", "PlayPause"),
+        ("STRING_UPARROW", ""),
+        ("STRING_DOWNARROW", ""),
+        ("STRING_LEFTARROW", ""),
+        ("STRING_RIGHTARROW", ""),
+        ("STRING_F1", ""),
+        ("STRING_F2", ""),
+        ("STRING_F3", ""),
+        ("STRING_F4", ""),
+        ("STRING_F5", ""),
+        ("STRING_F6", ""),
+        ("STRING_F7", ""),
+        ("STRING_F8", ""),
+        ("STRING_F9", ""),
+        ("STRING_F10", ""),
+        ("STRING_F11", ""),
+        ("STRING_F12", ""),
+        ("STRING_F13", ""),
+        ("STRING_F14", ""),
+        ("STRING_F15", ""),
+        ("STRING_F16", ""),
+        ("STRING_F17", ""),
+        ("STRING_F18", ""),
+        ("STRING_F19", ""),
+        ("STRING_F20", ""),
+        ("STRING_F21", ""),
+        ("STRING_F22", ""),
+        ("STRING_F23", ""),
+        ("STRING_F24", ""),
+        ("STRING_F25", ""),
+        ("STRING_F26", ""),
+        ("STRING_F27", ""),
+        ("STRING_F28", ""),
+        ("STRING_F29", ""),
+        ("STRING_F30", ""),
+        ("STRING_F31", ""),
+        ("STRING_F32", ""),
+        ("STRING_F33", ""),
+        ("STRING_F34", ""),
+        ("STRING_F35", ""),
+        ("STRING_INSERT", ""),
+        ("STRING_DELETE", ""),
+        ("STRING_HOME", ""),
+        ("STRING_BEGIN", ""),
+        ("STRING_END", ""),
+        ("STRING_PAGEUP", ""),
+        ("STRING_PAGEDOWN", ""),
+        ("STRING_PRINTSCREEN", ""),
+        ("STRING_SCROLLLOCK", ""),
+        ("STRING_PAUSE", ""),
+        ("STRING_SYSREQ", ""),
+        ("STRING_BREAK", ""),
+        ("STRING_RESET", ""),
+        ("STRING_STOP", ""),
+        ("STRING_MENU", ""),
+        ("STRING_USER", ""),
+        ("STRING_SYSTEM", ""),
+        ("STRING_PRINT", ""),
+        ("STRING_CLEARLINE", ""),
+        ("STRING_CLEARDISPLAY", ""),
+        ("STRING_INSERTLINE", ""),
+        ("STRING_DELETELINE", ""),
+        ("STRING_INSERTCHAR", ""),
+        ("STRING_DELETECHAR", ""),
+        ("STRING_PREV", ""),
+        ("STRING_NEXT", ""),
+        ("STRING_SELECT", ""),
+        ("STRING_EXECUTE", ""),
+        ("STRING_UNDO", ""),
+        ("STRING_REDO", ""),
+        ("STRING_FIND", ""),
+        ("STRING_HELP", ""),
+        ("STRING_MODESWITCH", ""),
```

Comment:
Just a nit: These look... interesting... perhaps escaping them and writing them numerically would be nicer?

Reply:
I agree; I copied them out of JPEXS and was too lazy to escape them :)